_includes/nav.html

```diff
@@ -1,13 +1,11 @@
 {% comment %} tag for this page is {% include nav.html %} {% endcomment %}
-<button style = "float:left" onclick = "openNav()" class="r3-button">Navigation Menu</button>
-<div>
-    <h1 id = "title">FRC 6321 Scouting</h1>
+
+<div class = 'navbar w3-bar' style='text-align: center'>
```

Comment:
This was very helpful, it does make the navbar look much better on the mobile version

Reply:
Do you think that it might be better to remove the "FRC 6321 Scouting" title at the top of the page and make the top bar stick to the very top of the page?